Makefile.include

```diff
@@ -30,8 +30,8 @@ include $(RIOTBASE)/Makefile.docker
 # Static code analysis tools provided by LLVM
 include $(RIOTBASE)/Makefile.scan-build
 
-# Path to the current directory relative to the git root
-BUILDRELPATH ?= $(shell git rev-parse --show-prefix)
+# Path to the current directory relative to RIOTBASE
+BUILDRELPATH ?= ${PWD:${RIOTBASE}/%=%}/
```

Comment:
This form is not bashism [1].

```
Substitution references (see Substitution References) are a simpler way to get the effect of the patsubst function:

$(var:pattern=replacement)
is equivalent to

$(patsubst pattern,replacement,$(var))
```

[1] https://www.gnu.org/savannah-checkouts/gnu/make/manual/html_node/Text-Functions.html